testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -961,3 +962,20 @@ def test_partially_missing_cryst():
     assert len(u.atoms) == 3
     assert len(u.trajectory) == 2
     assert_array_almost_equal(u.dimensions, 0.0)
+
+
+def test_cryst_em_warning():
+    #issue 2599  
+    with pytest.warns(UserWarning) as record:
+        u = mda.Universe(PDB_CRYOEM_BOX)
+    assert record[0].message.args[0] == "1 A^3 CRYST1 record," \
+                                        " this is usually a placeholder in" \
+                                        " cryo-em structures. Unit cell" \
+                                        " dimensions will not be set."
+                                    
+
+def test_cryst_em_select():
+    #issue 2599
+    u = mda.Universe(PDB_CRYOEM_BOX)
+    cur_sele=u.select_atoms('around 0.1 (resid 4 and name CA and segid A)')
```

Comment:
PEP8: As per my previous review, you'll want spaces around the `=` here.

Reply:
Is there anything more that I must do ? Thanks.

Reply:
So sorry for the delay here, the PR just got lost in a sea of GSOC related things. I'll try to re-review by the end of the day.